packages/libs/client/src/pact.ts

```diff
@@ -214,7 +214,7 @@ export class PactCommand
     // convert to IUnsignedTransaction
     const command: ICommand = {
       hash,
-      sigs: this.sigs.map((s) => (s === undefined ? { sig: '' } : s)),
+      sigs: this.sigs.map((s) => (s === undefined ? { sig: null } : s)),
```

Comment:
is it possible to receive `sigs: [null]`? Should we in that case check `!s` instead of `s === undefined`?